dynawo/sources/ModelicaCompiler/DYNCompileModelicaModel.cpp

```diff
@@ -450,16 +450,17 @@ compileLib(const string& modelName, const string& compilationDir) {
 }
 
 bool verifySharedObject(const string& library) {
-  // dlopen include <dlfcn.h>: to see if a shared object file
-  const char* filename = library.c_str();
-  void *handle;
-  handle = dlopen(filename, RTLD_NOW);
-  if (!handle) {
-    fprintf(stderr, "%s\n", dlerror());
-    printf(" DYNCompileModelicaModel: could not open .so by dlopen.");
+  try {
+    boost::dll::shared_library lib(library);
+    (void)lib;
```

Comment:
static_cast<void>

Reply:
For a statement whose purpose is to prevent the "unused" warning, I don't think it is relevant to use the static_cast

Reply:
It is just better to use a C++ style cast in my opinion

Reply:
OK